### DISCUSSION TITLE
Deleting a view

### DISCUSSION THREAD

**Discussion by agruenberg** [Budibase/budibase#8086] [Help] [answered]
I created a view but I don't need it anymore. And I cannot find any option to delete this view.

**Comment by andz-bb** [comment, marked as answer]
Hey @agruenberg 

when you hover over the view in your datasource list you should be able to click on the three dots and delete it from there.

<img width="472" alt="Screenshot 2022-10-03 at 10 18 34" src="https://user-images.githubusercontent.com/110921612/193543286-390ad4c1-539e-4a57-92db-2517b1e58e54.png">

Let me know if you're still having issues!

**Reply by agruenberg** [reply]
OMG, I'm blind. Sorry for that and thank you very much!

**Reply by andz-bb** [reply]
no problem at all, happy to help!
